var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -90,6 +90,7 @@ class Opencv(CMakePackage):
     depends_on('qt', when='+qt')
     depends_on('java', when='+java')
     depends_on('py-numpy', when='+python', type=('build', 'run'))
+    depends_on('protobuf@3.1.0', when='@3.3.0:')
```

Comment:
Should this be dependent on `protobuf@3.1.0` or `protobuf@3.1.0:`?

Reply:
Yes, their code does not compile with later versions of protobuf.